Cargo.toml

```diff
@@ -20,15 +20,15 @@ keywords = ["lancedb", "lance", "database", "vector", "search"]
 categories = ["database-implementations"]
 
 [workspace.dependencies]
-lance = { "version" = "=0.18.3", "features" = [
+lance = { "version" = "=0.19.1", "features" = [
     "dynamodb",
-], git = "https://github.com/lancedb/lance.git", tag = "v0.18.3-beta.2" }
-lance-index = { "version" = "=0.18.3", git = "https://github.com/lancedb/lance.git", tag = "v0.18.3-beta.2" }
-lance-linalg = { "version" = "=0.18.3", git = "https://github.com/lancedb/lance.git", tag = "v0.18.3-beta.2" }
-lance-table = { "version" = "=0.18.3", git = "https://github.com/lancedb/lance.git", tag = "v0.18.3-beta.2" }
-lance-testing = { "version" = "=0.18.3", git = "https://github.com/lancedb/lance.git", tag = "v0.18.3-beta.2" }
-lance-datafusion = { "version" = "=0.18.3", git = "https://github.com/lancedb/lance.git", tag = "v0.18.3-beta.2" }
-lance-encoding = { "version" = "=0.18.3", git = "https://github.com/lancedb/lance.git", tag = "v0.18.3-beta.2" }
+], git = "https://github.com/lancedb/lance.git", tag = "v0.19.1" }
+lance-index = { "version" = "=0.19.1", git = "https://github.com/lancedb/lance.git", tag = "v0.19.1" }
+lance-linalg = { "version" = "=0.19.1", git = "https://github.com/lancedb/lance.git", tag = "v0.19.1" }
+lance-table = { "version" = "=0.19.1", git = "https://github.com/lancedb/lance.git", tag = "v0.19.1" }
+lance-testing = { "version" = "=0.19.1", git = "https://github.com/lancedb/lance.git", tag = "v0.19.1" }
+lance-datafusion = { "version" = "=0.19.1", git = "https://github.com/lancedb/lance.git", tag = "v0.19.1" }
+lance-encoding = { "version" = "=0.19.1", git = "https://github.com/lancedb/lance.git", tag = "v0.19.1" }
```

Comment:
When we upgrade to stable versions (non-beta), we should reference the cargo released artifacts, not the git repo.

So:

```toml
lance = "=0.19.1"
...
```

Reply:
will change it to upgrade to v0.19.2-beta1 for `FTS search on multiple columns` feature